numpy/lib/histograms.py

```diff
@@ -200,16 +236,17 @@ def _hist_bin_auto(x):
     --------
     _hist_bin_fd, _hist_bin_sturges
     """
-    fd_bw = _hist_bin_fd(x)
-    sturges_bw = _hist_bin_sturges(x)
+    fd_bw = _hist_bin_fd(x, _)
+    sturges_bw = _hist_bin_sturges(x, _)
```

Comment:
Passing on unnamed arguments isn't a great idea -- that's a convention used for arguments that get thrown away. I would just call this `range` in the function signature.